frontend/src/views/LandingPage.vue

```diff
@@ -28,6 +28,29 @@
         />
       </b-card-group>
     </b-container>
+    <b-container>
```

Comment:
Please extract this container to a seperate component and add i18n for text labels.